src/model/mod.rs

```diff
@@ -12,9 +14,15 @@ pub const OAM_VERSION: &str = "core.oam.dev/v1beta1";
 // NOTE(thomastaylor312): If we ever end up supporting more than one kind, we should use an enum for
 // this
 pub const APPLICATION_KIND: &str = "Application";
+/// The version key, as predefined by the [OAM
+/// spec](https://github.com/oam-dev/spec/blob/master/metadata.md#annotations-format)
```

Comment:
just double checking, the line break here preserves the link?

Reply:
Yeah, it does as far as I know. This is just my wrapper thing